packages/react/src/components/select/select.tsx

```diff
@@ -0,0 +1,68 @@
+import { c, classy, m, SelectProps as BaseProps } from '@onfido/castor';
+import { Icon, useField } from '@onfido/castor-react';
+import React, { useState } from 'react';
+import { withRef } from '../../utils';
+
+let idCount = 0;
+
+/**
+ * `Select` uses an `Icon` that requires `Icons` (SVG sprite) to be included in
+ * your app.
+ *
+ * https://github.com/onfido/castor-icons#use-with-plain-code
+ */
+export const Select = withRef(
+  (
+    {
+      id = `castor_select_${++idCount}`,
+      placeholder,
+      defaultValue,
+      value,
+      borderless,
+      invalid,
+      children,
+      className,
+      onChange,
+      ...restProps
+    }: SelectProps,
+    ref: SelectProps['ref']
+  ): JSX.Element => {
+    const [empty, setEmpty] = useState(!!placeholder);
+    const { disabled, touched } = useField();
+
+    return (
+      <div className={classy(c('select'), m({ borderless }))}>
+        <select
+          disabled={disabled} // will be overriden by props if set
+          {...restProps}
+          ref={ref}
+          id={id}
+          defaultValue={
+            defaultValue || value != null ? undefined : placeholder && ''
```

Comment:
I think this is missing a parens, otherwise `defaultValue` would never apply.

```suggestion
            defaultValue || (value != null ? undefined : placeholder && '')
```

Reply:
Actually this one might read slightly better even

```suggestion
            defaultValue || (value == null && placeholder ? '' : undefined)
```